lms/static/scripts/frontend_apps/api-types.ts

```diff
@@ -4,21 +4,25 @@
  */
 import type { APICallInfo } from './config';
 
-/**
- * Object representing a file or folder resource in the LMS's file storage.
- */
```

Comment:
Can you update rather than remove this comment. Something like "Object representing a file, folder or document stored by the LMS".